src/proofPoints.ts

```diff
@@ -64,17 +69,182 @@ interface ProofPointValidateResult {
     statusMessage: string | null;
 }
 
+/**
+ * Proof point event type, the type of an {@link ProofPointEvent}
+ */
+enum ProofPointEventType {
+    Issued,
```

Comment:
All these TS files are using 4 space indent. https://standardjs.com/rules.html#rules suggests 2 space is the JS convention and all our other JS code uses 2 spaces. Can we update this to be consistent please?

Reply:
Done